pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -553,3 +564,22 @@ const storeElasticsearchTemplate = `{{- define "storeElasticsearch" }}
 	</buffer>
 </store>
 {{- end}}`
+
+const storeSyslogTemplate = `{{- define "storeSyslog" }}
+<store>
+	@type {{.RSyslogType}}
+	@id {{.StoreID}}
+	remote_syslog {{.Host}}
+	port {{.Port}}
+	hostname ${hostname}
+	facility {{.RSyslogFacility}}
```

Comment:
'user' - explicitly set based on https://github.com/openshift/origin-aggregated-logging/blob/release-3.11/fluentd/out_syslog_buffered.rb#L13-L21

same with all below

Reply:
Changed, but can you please clarify this for me - I though that [generate_syslog_config.rb ](https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/generate_syslog_config.rb#L16-L34) is executed before and sets the defaults so the plugin's defaults are not being applied, and therefore the defaults I see in the [4.1 documentation](https://docs.openshift.com/container-platform/4.1/logging/config/efk-logging-external.html#efk-logging-external-syslog_efk-logging-external) - how come that eventually the plugin's defaults are set?

Reply:
> Changed, but can you please clarify this for me - I though that [generate_syslog_config.rb ](https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/generate_syslog_config.rb#L16-L34) is executed before and sets the defaults so the plugin's defaults are not being applied, and therefore the defaults I see in the [4.1 documentation](https://docs.openshift.com/container-platform/4.1/logging/config/efk-logging-external.html#efk-logging-external-syslog_efk-logging-external) - how come that eventually the plugin's defaults are set?

Any config generation based on scripts and ruby files that are in the fluent image are no longer relevant; any reference provided to you is exactly that, a references for modifying it in LF.  At the heart of it, LogForwarding is a config generator for the collector and does not allow for any other tweaks other then what is possible via the LF api

Reply:
Oh, I understand this part - that it's no longer relevant since introducing LF. It's just that you pointed me to the defaults in 3.11 that are determined by out_syslog_buffered.rb so I wondered if in 4.1 and below those default were ever applied in light of the configuration generated by generate_syslog_config.rb

Reply:
I pointed you to the defaults so you had a value to use in place of the templated values.  If there are others which you believe are more appropriate please use your best judgement